weather_mv/loader_pipeline/ee.py

```diff
@@ -788,6 +801,10 @@ def start_ingestion(self, asset_data: AssetData) -> t.Optional[str]:
                 logger.info(f"Failed to create asset '{asset_name}' in earth engine: {e}. Moving on...")
                 return ""
```

Comment:
This is a bit problematic. We need to find a better way to handle this, instead of returning an empty string.

**REASON:**
In case the user wants to log the metrics, using `use_metrics` / `use_monitoring_metrics`, the flow will keep on working. Even for failed EE ingestions, we will get the metrics for them logged.